### PR TITLE
Update status of Ticket Mutation

### PR DESCRIPTION
**Pull request by abdul-shajin**
New Migration file for status in tickets table
New enum type for ticket status
Added mutation for changing ticket status
Added Multiple Mutation tests

### CODE REVIEW

File: app/graphql/mutations/change_ticket_status_mutation.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+    return_field :result, !types.Boolean
+    return_field :errors, types[Types::ErrorType]
+
+    resolve ->(object, inputs, context) {
+
+      ticket = Ticket.find_by(id: inputs[:ticket_id])
+
+      if ticket
+        result = ticket.update_attributes(status: inputs[:status])
```
Comment (neerajdotname) [inline app/graphql/mutations/change_ticket_status_mutation.rb]: `result` is not a good name. Change it to `updated_ticket`.

@abdul-shajin _a

File: db/schema.rb [hunk 2]
```diff
@@ -89,6 +89,7 @@
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.uuid "organization_id"
+    t.string "status"
```
Comment (neerajdotname) [inline db/schema.rb]: Please make it `null: false`.

Comment (abdul-shajin) [reply db/schema.rb]: Added default: 'open' too.

**Comment by neerajdotname** [comment]
@abdul-shajin _a please resolve the conflict and then you can merge the PR.

File: app/graphql/mutations/change_ticket_status_mutation.rb [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class Mutations::ChangeTicketStatusMutation
+  Perform = GraphQL::Relay::Mutation.define do
+    name "ChangeTicketStatus"
+
+    input_field :status, !Types::TicketStatusEnumType
+    input_field :ticket_id, !types.String
+
+    return_field :updated_ticket, !types.Boolean
```
Comment (neerajdotname) [inline app/graphql/mutations/change_ticket_status_mutation.rb]: Please return the field `ticket`. And type should be `Types::TicketType`.

@abdul-shajin _a

**Comment by neerajdotname** [comment]
@abdul-shajin _a Looks good. Go ahead and resolve the conflict and then merge the PR.

**Pull request merged**